algorithms/glued_trees/glued_trees.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.        A = -np.array(linalg.sqrtm(A))
</pre><p>this is no longer A, but A_sqrt</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>